lib/cocina/models/admin_policy.rb

```diff
@@ -4,74 +4,8 @@ module Cocina
   module Models
     # An admin policy object.
     class AdminPolicy < Struct
-      include Checkable
-
-      TYPES = [
```

Comment:
I don't think `TYPES` or `Checkable` should be part of the AdminPolicyAttributes, nor should the nested classes.